plugins/network-elements/nuage-vsp/src/com/cloud/network/element/NuageVspElement.java

```diff
@@ -303,13 +414,17 @@ protected boolean canHandle(Network network, Service service) {
         }
 
         if (!_networkModel.isProviderForNetwork(getProvider(), network.getId())) {
-            s_logger.debug("NuageElement is not a provider for network " + network.getDisplayText());
+            if (s_logger.isDebugEnabled()) {
+                s_logger.debug("NuageVsp is not a provider for network " + network.getDisplayText());
+            }
             return false;
         }
 
         if (service != null) {
             if (!_ntwkSrvcDao.canProviderSupportServiceInNetwork(network.getId(), service, getProvider())) {
-                s_logger.debug("NuageElement can't provide the " + service.getName() + " service on network " + network.getDisplayText());
+                if (s_logger.isDebugEnabled()) {
+                    s_logger.debug("NuageVsp can't provide the " + service.getName() + " service on network " + network.getDisplayText());
```

Comment:
This code is just checking whether the NuageVsp provider supports the services enabled on the given network, no need for specific information about the Nuage VSP.